packages/httpsPortal/src/httpsPortal.ts

```diff
@@ -88,6 +95,7 @@ export class HttpsPortal {
    * Remove an internal container from being exposed to the external internet
    */
   async removeMapping(mapping: HttpsPortalMapping): Promise<void> {
+    if (!(await this.isRunningHttps())) return;
```

Comment:
Shouldn't we throw an error here?

Reply:
probably although it would require a deeper research, I will open an issue so we can push this for this release